documentation/cluster-controller/cluster-controller.adoc

```diff
@@ -0,0 +1,267 @@
+= Cluster Controller
+
+The cluster controller is in charge to deploy a Kafka cluster alongside a Zookeeper ensemble. It's also able to deploy a 
```

Comment:
_in charge to deploy_ sounds strange ... wouldn't _in charge of deploying_ be better?